crates/biome_js_analyze/tests/specs/suspicious/useIterableCallbackReturn/invalidCheckForEach.js

```diff
@@ -0,0 +1,26 @@
+// These forEach cases should be invalid when checkForEach is true
```

Comment:
Should start with the magic comment `/* should generate diagnostics */` so our infra knows and can assert for it